include/ua_plugin_nodestore.h

```diff
@@ -70,7 +70,7 @@ typedef struct {
     UA_ExpandedNodeId *targetIds;
 } UA_NodeReferenceKind;
 
-#define UA_NODE_BASEATTRIBUTES                  \
+#define __UA_NODE_BASEATTRIBUTES                \
```

Comment:
The counter is added to all nodes.
But only variable nodes with a datasource are considered for the callback.

- The counter and the callback should appear always in pairs.
- In the future we might want to use the same mechanism for non-variable nodes as well. For example for event monitoring on objects or for other node attributes (e.g. user access rights that are checked with other systems in the background)

Can we put counter + callback in every node?
And count any type of monitoreditem, not only the value.

Reply:
I thought about adding a more generic callback to the server config. It could be triggered on every subscription with the node id or the node itself as identification.. the data source should keep that "opt in" callback tough.